packages/payment-integration-api/src/payment/payment.ts

```diff
@@ -65,6 +66,22 @@ export interface CreditCardInstrument {
     browser_info?: BrowserInfo;
 }
 
+export interface UsBankAccountInstrument {
```

Comment:
Wanted to understand what's the reasoning behind `Us` prefix here?

Reply:
@animesh1987 I renamed it to **WithBankAccountInstrument** following your example.